app/services/metrics/content_per_level_metric.rb

```diff
@@ -1,13 +1,26 @@
 module Metrics
   class ContentPerLevelMetric
```

Comment:
Maybe changing this class would be helpful, now that its about a couple of sets of metrics. Something like ContentDistributionMetrics?